python/lsst/ctrl/provenance/dc3.py

```diff
@@ -155,7 +159,7 @@ def queryRunOffset(self):
         self._globalDb.condParamString("runId", self._runid)
         self._globalDb.setQueryWhere("runId = :runId")
         self._globalDb.query()
-        if not self._globalDb.next() or self._globalDb.columnIsNull(0):
+        if not next(self._globalDb) or self._globalDb.columnIsNull(0):
```

Comment:
PS @srp3rd I tried running the `daf_persistence` tests on `lsst-dev` and they can't connect to the server either so I'm guessing the server is down?